rbi/core/file_test.rbi

```diff
@@ -1,4 +1,88 @@
 # typed: __STDLIB_INTERNAL
 
 module FileTest
+  # The file_name parameter for all of these methods can either be a String
+  # or an IO object.
+  FileName = T.type_alias(T.any(String, IO))
```

Comment:
Could you please write it explicitly in the methods? We're trying to optimize for readers.

Reply:
👍 done